elastic/kibana
# [Fleet] Make package policy name unique when copied multiple time

## Summary

Resolve #122126

Fix the copy function to increment the `(copy)` suffix when the same policy is copied multiple time.

The code had some complexity but I think the integration test should cover this well.

## How to test

1. Create a policy with a package policy 
2. Copy that policy a first time you should see the package policy with the name `* (copy)`
3. Copy that policy a second time (you need to change the name) you should see the package policy with the name `* (copy 2)`
